go/cmd/dolt/commands/diff.go

```diff
@@ -603,17 +615,15 @@ func diffUserTable(
 
 func diffDoltSchemasTable(
 	sqlCtx *sql.Context,
-	td diff.TableDelta,
 	sqlEng *engine.SqlEngine,
 	dArgs *diffArgs,
 	dw diffWriter,
 ) errhand.VerboseError {
-	err := dw.BeginTable(sqlCtx, td)
-	if err != nil {
-		return errhand.VerboseErrorFromError(err)
-	}
-
-	query := fmt.Sprintf("select from_fragment,to_fragment from dolt_diff('%s','%s','%s')", dArgs.fromRef, dArgs.toRef, doltdb.SchemasTableName)
+	// TODO: does this work when the table has been deleted in the current revision?
```

Comment:
More context on this comment. Why would the current revision have any impact on this. fromRef and toRef should work regardless of what HEAD is, yes? Also, If you deleted the table in the toRef (HEAD or otherwise) it works. There is a test for the dolt_diff method which shows this.

Reply:
I meant the `to` revision, but it as a just a note to myself, I'll delete